packages/framework/src/Resources/views/Mail/Order/withdrawalAdminMail.html.twig

```diff
@@ -0,0 +1,27 @@
+<h1>{{ 'New Withdrawal Request'|trans }}</h1>
+
+<p>{{ 'A customer has requested withdrawal from contract for the following order'|trans }}</p>
+
+<h2>{{ 'Order information'|trans }}</h2>
+<ul>
+    <li><strong>{{ 'Order number'|trans }}:</strong> {{ order.number }}</li>
+    <li><strong>{{ 'Customer email'|trans }}:</strong> {{ order.email }}</li>
+    <li><strong>{{ 'Order date'|trans }}:</strong> {{ order.createdAt|formatDateTime }}</li>
```

Comment:
should it be formatted for the timezone of the order domain?

Reply:
I am not sure, I do not think so. It is formatted for the admin timezone, which seems to be the best option to me, or? We can talk about it :slightly_smiling_face:

Reply:
Anyway, I have added some improvements to the template in 3897261ea4 and 7d82e5a31f